Userland/Libraries/LibJS/Heap/HeapBlock.cpp

```diff
@@ -18,6 +18,13 @@
 
 namespace JS {
 
+static size_t system_page_size()
+{
+    return sysconf(_SC_PAGESIZE);
+}
+
+size_t HeapBlockBase::block_size = system_page_size();
```

Comment:
Funnily enough, writing PAGE_SIZE here is equivalent

https://github.com/LadybirdBrowser/ladybird/blob/3e92ec80f3873bb81cbc8b0dc93bb3a6a145172c/AK/Platform.h#L275